gtk/rgrepositorywin.h

```diff
@@ -31,12 +31,6 @@
 #include "rsources.h"
 #include "rggtkbuilderwindow.h"
 
-#if !GTK_CHECK_VERSION(3,0,0)
- #include "rgvendorswindow.h"
-#else
- #warning "no rgvendorswindow in gtk3"
```

Comment:
We probably want to keep this too or even make it an error - the whole rgrepositorywin.h is not needed for ubuntu/debian iirc, it's a feature of apt-rpm which never got added to the deb based apt. So we could in a followup remove the entire thing or at least mark it very clearly as deprecated. I will merge your PR anyway because the rest looks very nice :)